js/pages/iptt_report/components/report/headerCells.js

```diff
@@ -15,6 +16,40 @@ const HeaderCell = ( props ) => {
     )
 }
 
+class LopActualHeaderCell extends React.Component {
+    constructor( props ) {
+        super(props);
+        this.style = props.styleWidth ? {
+            minWidth: `${props.styleWidth}px`
```

Comment:
Do we do a lot of direct styling with our react components?  Would it be better to add css classes for this, just to be consistent with the way to style other react components?